lib/printer.js

```diff
@@ -60,10 +60,13 @@ function getDefaultPrinterName() {
 
   // seems correct posix behaviour
   var printers= getPrinters();
-  for(i in printers) {
-    var printer = printers[i];
-    if(printer.isDefault === true) {
-      return printer.name;
+  if(printers && printers.length){
+    var i = printers.length;
```

Comment:
It is enough to declare just the variable. printers.length is useless because it will overwritten. I will correct it.